website/package.json

```diff
@@ -4,48 +4,44 @@
   "version": "1.0.0",
   "author": "HashiCorp",
   "dependencies": {
-    "@hashicorp/nextjs-scripts": "11.0.2",
-    "@hashicorp/react-button": "2.2.1",
-    "@hashicorp/react-code-block": "1.2.7",
-    "@hashicorp/react-content": "4.0.0",
-    "@hashicorp/react-docs-page": "4.0.0",
-    "@hashicorp/react-docs-sidenav": "3.2.5",
-    "@hashicorp/react-global-styles": "4.4.0",
-    "@hashicorp/react-hashi-stack-menu": "^1.0.8",
-    "@hashicorp/react-head": "1.1.1",
-    "@hashicorp/react-image": "2.0.1",
-    "@hashicorp/react-product-downloader": "4.0.2",
-    "@hashicorp/react-search": "^2.1.0",
-    "@hashicorp/react-section-header": "2.0.0",
-    "@hashicorp/react-subnav": "3.2.3",
-    "@hashicorp/react-vertical-text-block-list": "2.0.1",
-    "babel-plugin-import-glob-array": "0.2.0",
-    "imagemin-mozjpeg": "9.0.0",
-    "imagemin-optipng": "8.0.0",
-    "imagemin-svgo": "8.0.0",
-    "next": "9.4.4",
+    "@hashicorp/mktg-global-styles": "2.0.0",
+    "@hashicorp/nextjs-scripts": "13.0.0-canary.1",
```

Comment:
should we bump this to `13.3.0` now?

Reply:
This canary is specfically for mdx remote usage. will promote to stable once we are implemented across all sites, since I don't want it to block other features short term